### PR TITLE
Fix website

### PR DESCRIPTION
**Pull request by vjeux**
This puts the images inside of the blog folder in the output so we can safely copy them over. And moves the images in /blog/img/ in the source control so it's easier to discover (instead of having to search many levels deep).

Should make the website work!

### CODE REVIEW

**Comment by vjeux** [comment]
@facebook-github-bot shipit

File: website/server/convert.js [hunk 1]
```diff
@@ -134,6 +140,7 @@ function execute() {
 
   var files = glob.sync(DOCS_MD_DIR + '**/*.*');
   files.forEach(function(file) {
+    console.log(file);
```
Comment (bestander) [inline website/server/convert.js]: needed?

**Comment by bestander** [comment]
Looks great!

File: website/server/convert.js (hunk 1 above)
Comment (vjeux) [inline website/server/convert.js]: crap, didn't push the update

**Comment by vjeux** [comment]
Remove console.log and update image.

**Comment by vjeux** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**